### PR TITLE
Wheels

### PR DESCRIPTION
**Pull request by phschiele**

### CODE REVIEW

**Comment by phschiele** [comment]
@SteveDiamond Wheels and tests are working here.

Two open points:
- [ ] Windows tests were failing for 3.6 - 3.8, as far as I could tell, there was some issue with BLAS/LAPACK. I have fixed this by bumping to scipy == 1.5 and numpy == 1.19. One could likely downgrade these dependencies again after fixing BLAS/LAPACK explicitly before.
- [x] The wheels are not linked against BLAS/LAPACK yet. This should be fixable by adding installation command to the builder environment: https://stackoverflow.com/questions/67343308/problems-building-a-python-c-pybind11-package-with-cbuildwheel-using-github

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          CIBW_BEFORE_ALL_LINUX: yum install openblas
+        uses: joerick/cibuildwheel@v2.3.1
+
+      - name: Build source
+        if: ${{'True' == 'True' && env.SINGLE_ACTION_CONFIG == 'True'}}
+        run: |
+          python setup.py sdist --dist-dir=wheelhouse
+
+      - name: Release to pypi
+        if: ${{env.DEPLOY == 'True'}}
```
Comment (SteveDiamond) [inline .github/workflows/build.yml]: shell: bash

Comment (phschiele) [reply .github/workflows/build.yml]: Added in c1e858d

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        with:
+          name: wheels
+          path: ./wheelhouse
+
+      - name: Release to pypi
+        if: ${{'True' == 'True'}}
+        shell: bash
+        run: |
+          python -m pip install --upgrade twine
+          twine check
```
Comment (phschiele) [inline .github/workflows/build.yml]:
```suggestion
          twine check wheelhouse/*
```

**Pull request merged**